build.sbt

```diff
@@ -2,32 +2,22 @@ name := "sbt-pom-reader"
 
 organization := "com.typesafe.sbt"
 
-sbtPlugin := true
+licenses += ("Apache-2.0", url("http://opensource.org/licenses/Apache-2.0"))
 
-publishMavenStyle := false
+sbtPlugin := true
 
 libraryDependencies ++= Dependencies.pluginDependencies
 
-git.baseVersion := "1.0"
```

Comment:
@jsuereth Just remembered I'd moved the line into `version.sbt` where it was set to `1.1`.